daemon/service.go

```diff
@@ -102,6 +102,8 @@ func (s *ServiceV2) selectPool(rctx context.Context, language, content, filename
 			return "", nil, err
 		}
 		language = lang
```

Comment:
Does this one need a `normalize` call too?

Reply:
This one will already have it normalized after `s.detectLanguage` that calls it underneath